<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-FO8G76I" name="test.controller.TestOwner" tests="8" failures="0" timestamp="13 四月 2020 08:32:09 GMT" time="4.295" errors="1">
  <testcase name="queryMyRanking" time="0.320" classname="test.controller.TestOwner"/>
  <testcase name="receiveUpgradeAward" time="0.336" classname="test.controller.TestOwner"/>
  <testcase name="userInformation" time="0.347" classname="test.controller.TestOwner">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at test.service.UserInformation.userInformation(UserInformation.java:39)
at test.controller.TestOwner.userInformation(TestOwner.java:15)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at test.TestngRun.main(TestngRun.java:16)
]]>
    </error>
  </testcase> <!-- userInformation -->
  <testcase name="leaderboardList" time="2.185" classname="test.controller.TestOwner"/>
  <testcase name="receiveSeasonSettlementAward" time="0.001" classname="test.controller.TestOwner"/>
  <testcase name="querySeasonSettlementRewards" time="0.370" classname="test.controller.TestOwner"/>
  <testcase name="querySeasonNumber" time="0.377" classname="test.controller.TestOwner"/>
  <testcase name="queryCurrentSeason" time="0.359" classname="test.controller.TestOwner"/>
</testsuite> <!-- test.controller.TestOwner -->
